plugin-cgroupv2/src/cgroupv2.rs

```diff
@@ -31,21 +46,27 @@ impl FromStr for CgroupV2Metric {
             time_used_tot: 0,
             time_used_user_mode: 0,
             time_used_system_mode: 0,
+            anon_used_mem: 0,
+            file_mem: 0,
+            kernel_mem: 0,
+            pagetables_mem: 0,
         };
+
         for line in s.lines() {
             let parts: Vec<&str> = line.split_ascii_whitespace().collect();
             if parts.len() >= 2 {
+                let value = parts[1]
+                    .parse::<u64>()
+                    .context(format!("ERROR : Parsing of value : {}", parts[1]))?;
```

Comment:
Use `with_context` instead of `context`.
Difference:
- `context` always generate the error message even if there's no error
`with_context` does nothing if there's no error -> more efficient